src/itwinai/torch/tuning.py

```diff
@@ -1,18 +1,44 @@
+# --------------------------------------------------------------------------------------
+# Part of the interTwin Project: https://www.intertwin.eu/
+#
+# Created by: Anna Lappe
+#
+# Credit:
+# - Anna Lappe <anna.elisa.lappe@cern.ch> - CERN
+# --------------------------------------------------------------------------------------
+
+from typing import Dict
+
 from ray.tune.schedulers import (
     AsyncHyperBandScheduler,
     HyperBandForBOHB,
     HyperBandScheduler,
     PopulationBasedTraining,
 )
 from ray.tune.schedulers.pb2 import PB2  # Population Based Bandits
+from ray.tune.search.ax import AxSearch
 from ray.tune.search.bayesopt import BayesOptSearch
 from ray.tune.search.bohb import TuneBOHB
+from ray.tune.search.hebo import HEBOSearch
 from ray.tune.search.hyperopt import HyperOptSearch
+from ray.tune.search.nevergrad import NevergradSearch
+from ray.tune.search.optuna import OptunaSearch
+from ray.tune.search.zoopt import ZOOptSearch
 
 
 def get_raytune_search_alg(
-    tune_config, seeds=False
-) -> TuneBOHB | BayesOptSearch | HyperOptSearch | None:
+    tune_config: Dict,
+) -> (
+    TuneBOHB
+    | BayesOptSearch
+    | HyperOptSearch
+    | AxSearch
+    | HEBOSearch
+    | NevergradSearch
+    | OptunaSearch
+    | ZOOptSearch
+    | None
+):
```

Comment:
It might be easier to annotate this as "Searcher" (the super class for the search algorithms" instead of mentioning every single one, but not something you have to change.

Reply:
I get what you mean! I actually like having them listed like this to quickly see which algorithms are implemented. Although, to be fair, it's not exactly hard to see from the code

Reply:
I think that either is okay. The good thing about keeping it verbose like this is that you can see all the implemented strategies, the downside is that you have to remember to update this if you add another strategy. I'd say just do whichever you prefer :)